Objects/tupleobject.c

```diff
@@ -135,6 +135,18 @@ PyTuple_New(Py_ssize_t size)
     return (PyObject *) op;
 }
 
+PyObject *
+_PyTuple_NewUnsafe(Py_ssize_t size)
```

Comment:
I'm not sure if `_PyTuple_NewNotInitialized` is better.

Reply:
I suggest the name `_PyTuple_NewUninitialized`.